src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,876 @@
+    Contract:          OptimismMintableERC20Factory - OP Mainnet
+    Chain ID:          10
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x000000000000000000000000e01efbeb1089d1d1db9c6c8b135c934c0734c846
+    Raw New Value:     0x0000000000000000000000005493f4677a186f64805fe7317d6993ba4863988f
+    Decoded Kind:      address
+    Decoded Old Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L61">0xE01efbeb1089D1d1dB9c6c8b135C934C0734c846</a>
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L39">0x5493f4677A186f64805fe7317D6993ba4863988F</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[37] -----
+    Who:               0x7A8Ed66B319911A0F3E7288BDdAB30d9c0C875c3
+    Contract:          SystemConfig
+    Chain ID:          1868
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x000000000000000000000000ab9d6cb7a427c0765163a7f45bb91cafe5f2d375
+    Raw New Value:     0x000000000000000000000000760c48c62a85045a6b69f07f4a9f22868659cbcc
+    Decoded Kind:      address
+    Decoded Old Value: 0xAB9d6cB7A427c0765163A7f45BB91cAfe5f2D375
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/validation/standard/standard-versions-mainnet.toml#L27">0x760C48C62A85045A6B69f07F4a9f22868659CbCc</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[38] -----
+    Who:               0x8062AbC286f5e7D9428a0Ccb9AbD71e50d93b935
+    Contract:          ProtocolVersions
+    Chain ID:          1
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000042f0bd8313ad456a38061308857b2383fe2c72a0
+    Raw New Value:     0x00000000000000000000000037e15e4d6dffa9e5e320ee1ec036922e563cb76c
+    Decoded Kind:      address
+    Decoded Old Value: 0x42F0bD8313ad456A38061308857b2383fe2c72a0
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/84bce73573f130008d84bae6e924163bab589a11/validation/standard/standard-versions-mainnet.toml#L42">0x37E15e4d6DFFa9e5E320Ee1eC036922E563CB76C</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
```

Comment:
Missing state diff for: 0x847B5c174615B1B7fDF77088225
![CleanShot 2025-03-26 at 18 06 14@2x](https://github.com/user-attachments/assets/f44ef2d5-6c8e-4e5f-bd6c-58a394f49613)
6e2D3E95b9D92

Reply:
Added here: https://github.com/ethereum-optimism/superchain-ops/blob/1f08f4346ed83db85df5687a62edec938c0ff4c4/src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md#L661

But IDK where to link it to. WDYT?

Reply:
What I ended up doing is not linking to the address anywhere, but using cast to get the address from the PAO:

https://github.com/ethereum-optimism/superchain-ops/blob/3e8f8f5c3413ac2843167668f633dfc8d504cbad/src/improvements/tasks/eth/000-opcm-upgrade-v200/VALIDATION.md#L818-L820